packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -2191,7 +2191,8 @@ class TextSelectionGestureDetectorBuilder {
             case PointerDeviceKind.trackpad:
             case PointerDeviceKind.stylus:
             case PointerDeviceKind.invertedStylus:
-              // Precise devices should place the cursor at a precise position.
+              // TODO(camsim99): Determine spell check toolbar behavior in these cases: https://github.com/flutter/flutter/issues/119573.
+              // Precise devices should place the cursor at a precise position if the word at the text position is not misspelled.
```

Comment:
Nit: Put the GitHub URL on its own line, and make sure the other lines aren't going beyond the character width limit.